### ISSUE TITLE
Merge r3.0 into master

### ISSUE THREAD

**Issue by sqs** [sourcegraph/syntaxhighlight#12]
@gbbr, I just updated branch r3.0 to compile against the latest nodb branch of go-sourcegraph. When that is merged, can you merge r3.0 into master?

**Comment by gbbr** [comment]
Thank you. Yes, of course

**Comment by shurcooL** [comment]
Note that the majority of r3.0 branch was already merged into master (rebased, with minor fixes) in #11.

So you'd only need to merge the new commits on [r3.0 branch](https://github.com/sourcegraph/syntaxhighlight/commits/r3.0): it currently appears to be a single commit 22d8200cdad3fcc3cd0de2550ec6cf482f8ba43e.

**Comment by shurcooL** [comment]
You can just cherry-pick it on top, it merges nicely (I tried locally).

**Comment by shurcooL** [comment]
I'll take care of this since I happened to have it done locally and it's easy. @gbbr said he doesn't mind.

**Comment by gbbr** [comment]
Of course not. Thank you!

**Comment by shurcooL** [comment]
Made #13 to close this issue. Once `nodb` branch of `go-sourcegraph` is merged, I'll test the PR if it needs updating, and merge it into `master`.
